modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -11,7 +11,7 @@
 The address pool custom resource definition (CRD) and API documented in "Load balancing with MetalLB" in {product-title} 4.10 can still be used in {product-version}. However, the enhanced functionality associated with advertising the `IPAddressPools` with layer 2 or the BGP protocol is not supported when using the address pool CRD.   
```

Comment:
functionality associated with advertising the ip address with Layer 2 (L2Advertisement) or the BGP (BGPAdvertisement) protocol is not supported when using 'Addresspool' CRD.

Reply:
Layer 2 and BGP protocol used extensively in these sections so I haven't expanded on those, but I did update the sentence to make it clearer based on your feedback. Hope it works.